src/components/TodoList.js

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import TodoItemHandler from './TodoItemHandler';
+
+function TodoList(props) {
+  if (!props.requested) {
+    return (
+      <div>
+        <button className="initialButton" type="button" title="click to load TODOS" onClick={props.buttonClick}>Load</button>
+      </div>
+    );
+  }
+  if (props.todos === null) {
+    return <div className="myPreloader"><p>Loading</p><div class="lds-ellipsis"><div></div><div></div><div></div><div></div></div></div>;
```

Comment:
try to formatting code something like this
```suggestion
    return (
    <div className="myPreloader">
      <p>Loading</p>
      <div class="lds-ellipsis">
        <div></div>
        <div></div>
        <div></div>
        <div></div>
      </div>
    </div>);
```